mender-update/standalone/standalone.cpp

```diff
@@ -195,6 +197,66 @@ error::Error RemoveStateData(database::KeyValueDatabase &db) {
 	return db.Remove(context::MenderContext::standalone_state_key);
 }
 
+static expected::expected<pair<http::ClientPtr, io::ReaderPtr>, error::Error>
+ClientAndReaderFromUrl(const string &src) {
+	http::ClientPtr http_client;
+	auto maybe_artifact_reader = events::io::ReaderFromAsyncReader::Construct(
```

Comment:
I'm not super-happy with how long this lamdba is, and I considered turning it into a reusable convenience function when using direct Readers. OTOH, I suspect we need to revisit this area when we start using the parser in daemon mode, and maybe this part will change, so maybe it's good enough for now.

I'll let you guys judge how bad it is.